message/messaging_service.hh

```diff
@@ -254,6 +255,7 @@ public:
             memory_config mcfg, scheduling_config scfg, bool sltba = false, bool listen_now = true);
     ~messaging_service();
 public:
+    future<> start_listen_ex();
```

Comment:
Can we avoid having two entry points? e.g. rename start_listen() to something private and internal?